pkg/overlay/mockoverlay.go

```diff
@@ -53,8 +53,9 @@ func (mo *MockOverlay) BulkLookup(ctx context.Context, reqs *pb.LookupRequests)
 	*pb.LookupResponses, error) {
 	var responses []*pb.LookupResponse
 	for _, r := range reqs.Lookuprequest {
-		n := *mo.nodes[r.NodeID]
-		resp := &pb.LookupResponse{Node: &n}
+		// NOTE (Dylan): tests did not catch missing node case, need updating
```

Comment:
Rather than adding a note, add a test case for this.

Reply:
Anyways, it looks like the mock overlay has no tests.

Reply:
I was going to add a test for it, but didn't for exactly that reason.